README.md

```diff
@@ -91,8 +98,24 @@ DATABASES['default']['NAME'] = 'var/workbench.db'
 ```
 
 Because this XBlock uses a Django model, you need to sync the database
-before starting the workbench. Run this from the XBlock repository
-root:
+before starting the workbench. `problem-builder` uses MySQL 5.6 and hence you can quickly spin up
```

Comment:
I think it would be clearer to say:

> Testing `problem-builder` in the workbench requires MySQL instead of the standard SQLite configuration it uses. You can quickly spin up an instance of MySQL with Docker using the following command:

Reply:
Thanks, I have updated it :+1:

Reply:
Thanks, yeah this makes it clearer, I have updated it :D